code/game/objects/items.dm

```diff
@@ -265,6 +272,17 @@ GLOBAL_VAR_INIT(embedpocalypse, FALSE) // if true, all items will be able to emb
 /obj/item/proc/suicide_act(mob/user)
 	return
 
+/// Checks if this atom uses the GAS system and if so updates the worn and inhand icons
+/obj/item/proc/update_item_greyscale()
+	if(greyscale_colors)
+		if(greyscale_config_worn)
+			worn_icon = SSgreyscale.GetColoredIconByType(greyscale_config_worn, greyscale_colors)
+		if(greyscale_config_inhand_left)
+			lefthand_file = SSgreyscale.GetColoredIconByType(greyscale_config_inhand_left, greyscale_colors)
+		if(greyscale_config_inhand_right)
+			righthand_file = SSgreyscale.GetColoredIconByType(greyscale_config_inhand_right, greyscale_colors)
+	return
```

Comment:
You want something more in line with this, to lower the amount of unneeded indents https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-early-return

Reply:
I think I got it, please check

Reply:
yeah that's just fine, good work